lib/mix/tasks/development_seeds.ex

```diff
@@ -32,4 +36,8 @@ defmodule Mix.Tasks.DevelopmentSeeds do
        weeklyinfo
      )
   end
+
+  defp print_user(user) do
```

Comment:
Just a thought:

You could change this to:

```elixir
def print(%User{}=user) do
  IO.puts "User: #{user.email}/#{user.password}"
end

build(:user) |> save |> print
```

Then as you add other models you want to output they all just use `print`